m4/odp_dpdk.m4

```diff
@@ -120,7 +120,7 @@ AS_IF([test "x$1" = "xsystem"], [dnl
     fi
     DPDK_LIB_PATH=`AS_DIRNAME(["$DPDK_LIB_PATH"])`
 ], [dnl
-    DPDK_CPPFLAGS="-isystem $1/include"
+    DPDK_CPPFLAGS="-isystem $1/include/dpdk"
```

Comment:
Should we then switch to including `<dpdk/rteFOO.h>` headers instead. This would allow us to simplify `DPDK_CPPFLAGS`.

Reply:
We could do that too, but we would still have to set the correct include path anyway, so it doesn't simplify `DPDK_CPPFLAGS` much.
DPDK's example code doesn't prefix the headers, though.

Reply:
It does not, as it is built with non-installed tree.
Another question, should we support non-installed DPDK tree?

Reply:
Good question. I think building against a non-installed tree is not right, but this change may break someone's workflow, which made me realize `scripts/build-pktio-dpdk` needed to be updated as well.

Reply:
./scripts/build-pktio-dpdk does make install T=${TARGET} EXTRA_CFLAGS="-fPIC".  Is it not enough for install?

Reply:
According to the DPDK documentation:
> install T=       configure, build and install a target in DESTDIR
I get an error if you don't provide `DESTDIR`, when doing so:
> Installation cannot run with T defined and DESTDIR undefined

`DESTDIR` may not be needed, but since it will try to install kernel modules, it is probably good to provide a staging directory somewhere where the user can write to.